server.go

```diff
@@ -200,6 +201,46 @@ func (d *Server) receiveUpload(
 	return true
 }
 
+func (d *Server) scorecheck(predictionfilepath string, resultfilepath string) (float32, error) {
+
+	predictionfile, err := os.Open(predictionfilepath)
+	if err != nil {
+		d.log.Printf("error: %q reading file", err)
+		return 0, err
+	}
+	defer predictionfile.Close()
+
+	resultfile, err := os.Open(resultfilepath)
+	if err != nil {
+		d.log.Printf("error: %q reading file", err)
+		return 0, err
+	}
+	defer resultfile.Close()
+
+	scanpredictionfile := bufio.NewScanner(predictionfile)
+	scanresultfile := bufio.NewScanner(resultfile)
+
+	totallines := float32(0)
```

Comment:
I think you can write:
```
totallines := 0.0
```
And it will infer it is a float. Or in this case even this is better:

```
var totallines float32
```